devops/arangodb.yml

```
@@ -4,7 +4,10 @@
 
   vars:
     ansible_python_interpreter: /usr/bin/python3
-    arangodb_password: "letmein"
+
+  vars_prompt:
+    - name: arangodb_root_password
+      prompt: "Password for ArangoDB root account"
 
```

We could also supply an override-able default if you think that might be useful?
 
```suggestion
      prompt: "Password for ArangoDB root account"
      default: "letmein"
```

---

Also, having this as a prompt means we'd never be running this in an automated way, right?

---

@JackWilb: No default--this forces the devops engineer to select and keep track of a password.

@AlmightyYakob: that's true, but you can avoid the prompt by using the `--extra-vars` option on the command line. If the README updates are not clear enough about that, let me know and I'll fix it.